docker-compose/docker-compose.yaml

```diff
@@ -1,23 +1,22 @@
-version: "3"
+version: "3.8"
 
 services:
   iroha:
-    image: hyperledger/iroha2:lts@sha256:4072d09bee376c8fcbc586d35ea25624738694ce2c37aa0db60c9998c613048b
+    image: iroha2/iroha2@sha256:4b61b866e15039989e689e3403986ebb207628c36d956d2dc99078e19764e212
```

Comment:
shouldn't that be smth like `docker.soramitsu.co.jp/iroha2/iroha2@sha256:4b61b866e15039989e689e3403986ebb207628c36d956d2dc99078e19764e212`
locally it works for you since you may be logged in to the repository (though this image does not require authorization)